### PR TITLE
implement validate function.

### PR DESCRIPTION
**Pull request by y-hiraoka**
https://github.com/editor-js/link/issues/36

Blocks data without link must not exists.
`validate` method will be called and remove them  when Editor.js is saving data.

### CODE REVIEW

**Comment by neSpecc** [comment]
Update a patch version pls

**Comment by y-hiraoka** [comment]
@neSpecc I did it.

File: src/index.js [hunk 1]
```diff
@@ -141,6 +141,18 @@ export default class LinkTool {
     return this.data;
   }
 
+  /**
+   * Validate Block data
+   * - check for emtiness
```
Comment (ranemihir) [inline src/index.js:146]:
```suggestion
   * - check if given link is an empty string or not.
```

File: src/index.js [hunk 2]
```diff
@@ -141,6 +141,18 @@ export default class LinkTool {
...
+   *
+   * @public
+   *
+   * @returns {boolean} false if saved data is not correct, otherwise true
```
Comment (ranemihir) [inline src/index.js:150]:
```suggestion
   * @returns {boolean} false if saved data is incorrect, otherwise true
```

**Review by ranemihir — APPROVED** [review]

**Comment by y-hiraoka** [comment]
@ranemihir Thanks for your suggestion.
I've fixed them.

**Review by robonetphy — APPROVED** [review]

**Pull request merged**